puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/bastion.yaml

```diff
@@ -8,6 +8,7 @@
           basedir: 'plugin'
           branches:
             - '${branch}'
```

Comment:
In foreman-rpm-packaging-pr we also changed this to `${ghprbActualCommit}`

Reply:
`test_foreman_javascript` isn't a GHPRB job itself, but it can sometimes get kicked by `test_bastion_pull_request`.